recipe/meta.yaml

```diff
@@ -1,16 +1,15 @@
 {% set name = "aom" %}
-{% set version = "3.9.1" %}
+{% set version = "3.12.0" %}
 
 package:
   name: {{ name|lower }}
   version: {{ version }}
 
 source:
   url: https://storage.googleapis.com/aom-releases/libaom-{{ version }}.tar.gz
-  sha256: dba99fc1c28aaade28dda59821166b2fa91c06162d1bc99fde0ddaad7cecc50e
+  sha256: d89daa160a0ea1409c4193be5b17c9591024c4f5a0e545dcb9d197535c66836e
   patches:
-    - 0001-Fix-compilation-flags-for-win32.patch
-    - 0002-Remove-all-reference-of-aom_static.patch
+    # - 0002-Remove-all-reference-of-aom_static.patch
```

Comment:
why does this not apply anymore?

Reply:
You should try to revive this patch. If you have tried and failed, I can try to look later.